v0.92-988/non-regression.sh

```diff
@@ -173,15 +198,8 @@ function jerasure_action() {
     non_regression $action "$@" || return 1
     if test "$action" = --check ; then
         path=$(ceph_erasure_code_non_regression --show-path "$@")
```

Comment:
And the --show-path and --path options of ceph/src/test/erasure-code/ceph_erasure_code_non_regression.cc can also be removed.